_includes/payee-reference.md

```diff
@@ -17,10 +17,17 @@ specific processing rules depending on specifications in the contract.
     delivery of a transactional operation from the merchant system.
 *   Its length and content validation is dependent on whether the
     `transaction.number` or the `payeeReference` is sent to the acquirer.
+        {% if api_resource == "card" %}
     1.  If you select *Option A* in the settlement process (Swedbank Pay will
-        handle the settlement), Swedbank Pay will send the `transaction.number` to
-        the acquirer and the `payeeReference` must be in the format of characters
-        `A-Za-z0-9` (including `-`) and `string(30)`.
+        handle the settlement), Swedbank Pay will send the `transaction.number`
+        to the acquirer and the `payeeReference` must be in the format of
+        characters `A-Za-z0-9` (including `-`) and `string(50)`.
+        {% else %}
+    1.  If you select *Option A* in the settlement process (Swedbank Pay will
```

Comment:
Codacy found an issue: [[ordered-list-marker-value] Marker should be `2`, was `1`](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6110913)